src/components/Login.vue

```diff
@@ -68,6 +80,7 @@
 
 <script>
 import inputValidation from '@/components/mixins/inputValidation'
+import listOfNodes from '@/mocks/nodes.json'
```

Comment:
I think we should move assets and nodes from "mocks" folder and put it to the `data` folder maybe? :)